### PR TITLE
WIP: Preparing completions towards borg 1.2

### PR DESCRIPTION
**Pull request by SanskritFritz**
Fish and bash completions updated to reflect current master state. Removed possibly misleading borg version information from the files.

### CODE REVIEW

File: scripts/shell_completions/bash/borg [hunk 1]
```diff
@@ -13,7 +13,7 @@ _borg()
 	local cur="${COMP_WORDS[COMP_CWORD]}"
 	local prev="${COMP_WORDS[COMP_CWORD-1]}"
 	local prevprev="${COMP_WORDS[COMP_CWORD-2]}"
-	local common_opts="-h --help --version --critical --error --warning --info -v --verbose --debug --debug-topic -p --progress --log-json --lock-wait --show-version --show-rc --umask --remote-path --remote-ratelimit --consider-part-files --debug-profile"
+	local common_opts="-h --help --version --critical --error --warning --info -v --verbose --debug --debug-topic -p --progress --log-json --lock-wait --show-version --show-rc --no-files-cache --umask --remote-path --remote-ratelimit --consider-part-files --debug-profile"
```
Comment (ThomasWaldmann) [inline scripts/shell_completions/bash/borg]: --no-files-cache? isn't that deprecated (replacement: --files-cache=disabled)?

File: scripts/shell_completions/bash/borg [hunk 2]
```diff
@@ -37,11 +37,6 @@ _borg()
 			COMPREPLY=( $(compgen -W "${encryption_modes}" -- ${cur}) )
 			return 0
 			;;
-		'--files-cache')
-			local files_cache_mode="ctime,size,inode mtime,size,inode ctime,size mtime,size rechunk,ctime rechunk,mtime disabled"
-			COMPREPLY=( $(compgen -W "${files_cache_mode}" -- ${cur}) )
-			return 0
-			;;
```
Comment (ThomasWaldmann) [inline scripts/shell_completions/bash/borg]: why remove?

File: scripts/shell_completions/bash/borg [hunk 3]
```diff
@@ -71,10 +66,10 @@ _borg()
 				local opts="-e --encryption --append-only --storage-quota ${common_opts}"
 				;;
 			*' create '*)
-				local opts="-n --dry-run -s --stats --list --filter --json --no-cache-sync --no-files-cache -e --exclude --exclude-from --pattern --patterns-from --exclude-caches --exclude-if-present --keep-exclude-tags --keep-tag-files --exclude-nodump -x --one-file-system --numeric-owner --noatime --noctime --nobsdflags --ignore-inode --mtime --files-cache --read-special --comment --timestamp -c --checkpoint-interval --chunker-params -C --compression ${common_opts}"
```
Comment (ThomasWaldmann) [inline scripts/shell_completions/bash/borg]: why remove?

File: scripts/shell_completions/bash/borg [hunk 4]
```diff
@@ -71,10 +66,10 @@ _borg()
...
+				local opts="-n --dry-run -s --stats --list --filter --json --no-cache-sync -e --exclude --exclude-from --pattern --patterns-from --exclude-caches --exclude-if-present --keep-exclude-tags --keep-tag-files -x --one-file-system --numeric-owner --noatime --noctime --ignore-inode --read-special --comment --timestamp -c --checkpoint-interval --chunker-params -C --compression ${common_opts}"
 				;;
 			*' extract '*)
-				local opts="--list -n --dry-run --numeric-owner --nobsdflags --stdout --sparse -e --exclude --exclude-from --pattern --patterns-from --strip-components ${common_opts}"
```
Comment (ThomasWaldmann) [inline scripts/shell_completions/bash/borg]: why remove?

**Comment by SanskritFritz** [comment]
OK, I'll file another PR in the next weeks, I found where I went wrong here. Closing.

**Pull request closed without merging**